src/sections/Meshery/Meshery-integrations/Individual-Integrations/relatedIntegration.js

```diff
@@ -91,11 +116,11 @@ const RelatedIntegration = ({ category }) => {
       <Slider {...settings}>
         {
           IntegrationList.map((item, index) => {
-            if (item.category === category && item.status != "InProgress"){
+            if (item.frontmatter.category === category && item.status != "InProgress"){
               return (
                 <div key={index}>
-                  <Link to={`/service-mesh-management/meshery/integrations/${item.name.toLowerCase()}`}>
-                    <img src={item.img} alt={item.name} height={60} className="img-effect" />
+                  <Link to={`/service-mesh-management/meshery/integrations/${item.frontmatter.title.toLowerCase().replaceAll(" ", "-")}`}>
```

Comment:
Fetch the slug in graphql and use that here instead of updating it using a hacky way.
i.e,
```
fields {
    slug
}
```